pkg/netbox/api/ip_address_claim_test.go

```diff
@@ -35,20 +35,33 @@ func TestIPAddressClaim(t *testing.T) {
 	mockIPAddress := mock_interfaces.NewMockIpamInterface(ctrl)
 	mockTenancy := mock_interfaces.NewMockTenancyInterface(ctrl)
 
-	parentPrefixId := int64(3)
-	parentPrefix := "10.114.0.0"
-	ipAddress1 := "10.112.140.1/24"
-	ipAddress2 := "10.112.140.2/24"
-	singleIpAddress2 := "10.112.140.2/32"
-
-	// example of available IP address
-	availAddress := func() []*netboxModels.AvailableIP {
+	// test data for IPv4 ip address claim
+	parentPrefixIdV4 := int64(3)
+	parentPrefixV4 := "10.114.0.0"
+	ipAddressV4_1 := "10.112.140.1/24"
+	ipAddressV4_2 := "10.112.140.2/24"
+	singleIpAddressV4_2 := "10.112.140.2/32"
+
+	// example of available IPv4 IP addresses
+	availAddressesIPv4 := func() []*netboxModels.AvailableIP {
 		return []*netboxModels.AvailableIP{
-			{Address: ipAddress1},
-			{Address: ipAddress2},
+			{
+				Address: ipAddressV4_1,
+				Family:  int64(4),
+			},
+			{
+				Address: ipAddressV4_2,
+				Family:  int64(4),
+			},
 		}
 	}
 
+	// test data for IPv6 ip address claim
+	parentPrefixIdV6 := int64(4)
+	parentPrefixV6 := "2001:db8:85a3:8d3::/64"
+	ipAddressV6 := "2001:db8:85a3:8d3::2/64"
```

Comment:
Question: is there a specific reason we don't do `ipAddressV6_1` and `ipAddressV6_2`, like we have for IPv4? :)

Reply:
For IPv4 there are some test cases that where the mock is set up to return two ip addresses and some where it only returns one. As the selection of the ip uses same function for IPv4 and IPv6, I didn't add additional tests where two IPs are returned for IPv6.

Reply:
Got it, thanks!